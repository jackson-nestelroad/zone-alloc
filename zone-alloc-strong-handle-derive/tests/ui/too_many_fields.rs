use zone_alloc::Handle;
use zone_alloc_strong_handle_derive::StrongHandle;

#[derive(StrongHandle)]
struct IntegerHandle(Handle, bool);

fn main() {}
