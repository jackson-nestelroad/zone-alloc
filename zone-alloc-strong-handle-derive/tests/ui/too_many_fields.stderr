error: input must have exactly one unnamed field
 --> tests/ui/too_many_fields.rs:4:10
  |
4 | #[derive(StrongHandle)]
  |          ^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `StrongHandle` (in Nightly builds, run with -Z macro-backtrace for more info)
